avalanche/benchmarks/utils/data_loader.py

```diff
@@ -93,12 +99,17 @@ def __init__(
             will not contribute to the minibatch composition near the end of
             the epoch.
         :param distributed_sampling: If True, apply the PyTorch
-            :class:`DistributedSampler`. Defaults to False.
+            :class:`DistributedSampler`. Defaults to True.
+            Note: the distributed sampler is not applied if not running
+            a distributed training, even when True is passed.
         :param never_ending: If True, this data loader will cycle indefinitely
             by iterating over all datasets again and again and the epoch will
             never end. In this case, the `termination_dataset` and
             `oversample_small_datasets` parameters are ignored. Defaults to
             False.
+        :param use_ffcv: If True, use FFCV data loading mechanism. Has effect
+            only if the support for FFCV has been explicitly enabled by the
```

Comment:
The general idea is to allow users to disable FFCV for specific experiences/phases if needed. They already have to enable the FFCV support in the main script, so having use_ffcv defaulting to True makes sense. However, by setting `use_ffcv` they can disable FFCV temporarily.

Reply:
Is there a use case where someone enables ffcv but only wants it for some experiences/phases? After all, the user explicitly passed a benchmark to `enable_ffcv`. It can still pass a normal benchmark to avoid using ffcv loaders

Reply:
For this first version, I can remove that flag. If users need it, we can add it later!